cmd/restic/cmd_stats.go

```diff
@@ -50,11 +50,26 @@ Exit status is 0 if the command was successful, and non-zero if there was any er
 	},
 }
 
+// StatsOptions collects all options for the stats command.
+type StatsOptions struct {
+	// the mode of counting to perform (see consts for available modes)
+	countMode string
+
+	// filter snapshots by, if given by user
+	Hosts []string
+	Tags  restic.TagLists
+	Paths []string
+}
+
+var statsOptions StatsOptions
+
 func init() {
 	cmdRoot.AddCommand(cmdStats)
 	f := cmdStats.Flags()
-	f.StringVar(&countMode, "mode", countModeRestoreSize, "counting mode: restore-size (default), files-by-contents, blobs-per-file, or raw-data")
-	f.StringArrayVarP(&snapshotByHosts, "host", "H", nil, "filter latest snapshot by this hostname (can be specified multiple times)")
+	f.StringVar(&statsOptions.countMode, "mode", countModeRestoreSize, "counting mode: restore-size (default), files-by-contents, blobs-per-file, or raw-data")
```

Comment:
Normally it's written without the last , (that is, "foo, bar, star or blah" instead of "foo, bar, star, or blah").